packages/angular/standalone/src/navigation/tabs.ts

```diff
@@ -11,11 +12,13 @@ import { IonRouterOutlet } from './router-outlet';
     <ng-content select="[slot=top]"></ng-content>
     <div class="tabs-inner" #tabsInner>
       <ion-router-outlet
+        *ngIf="tabs?.length === 0"
```

Comment:
Similar feedback here if `tabs` can be undefined.

Reply:
https://github.com/ionic-team/ionic-framework/pull/29786/commits/f6132ca59a65c55f285ed0aad7f5e45a8e1cdc91